### PR TITLE
adding basic filtering, fixing artist link issue

### PR DESCRIPTION
**Pull request by ashkan18**
Added filtering checkboxes for being able to filter subscription/follow artist events.
Fixed artist link issue

screen shot 2016-06-16 at 4 50 16 pm

note in 👆 that filtering doesn't remove existing events from page it just stops adding new events of that type.

### CODE REVIEW

**Comment by ashkan18** [comment]
@joeyAghion I remember you brought this up in our stand up this morning, feel free to not merge this and write/experience your approach.

File: web/static/css/app.css [hunk 1]
```diff
@@ -32,6 +32,11 @@ form.link, form.button {
   background-size: 100px 35px;
 }
 
+.filter-seciton {
```
Comment (joeyAghion) [inline web/static/css/app.css]: typo

File: web/static/js/socket.js [hunk 2]
```diff
@@ -54,15 +54,19 @@ let socket = new Socket("/socket", {params: {token: window.userToken}})
 socket.connect()
 let moment = require('moment')
 let messagesContainer = $("#messages")
+let artistFollowCheckbox = $('#show-follow-artist')
+let subscriptionCheckbox = $('#show-subscritpions')
```
Comment (joeyAghion) [inline web/static/js/socket.js]: typo

File: web/static/js/socket.js [hunk 3]
```diff
@@ -54,15 +54,19 @@ let socket = new Socket("/socket", {params: {token: window.userToken}})
...
 // Now that you are connected, you can join channels with a topic:
 let subscriptionChannel = socket.channel("subscriptions", {})
 subscriptionChannel.join()
   .receive("ok", resp => { console.log("Joined successfully", resp) })
   .receive("error", resp => { console.log("Unable to join", resp) })
 subscriptionChannel.on("activated", payload => {
-  let newItem = $(`<li class="news-item">${moment().format("LT")}: ${payload.subject.display}</a> ${payload.verb} ${payload.object.root_type} for <a href=${payload.properties.partner.id}>${payload.properties.partner.name}</a></li>`)
-  newItem.prependTo(messages).hide().slideDown()
+  if (subscriptionCheckbox.is(':checked')) {
+    let newItem = $(`<li class="news-item">${moment().format("LT")}: ${payload.subject.display}</a> ${payload.verb} ${payload.object.root_type} for <a href=${payload.properties.partner.id}>${payload.properties.partner.name}</a></li>`)
```
Comment (joeyAghion) [inline web/static/js/socket.js:67]: Reading about [template literals...](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).

**Comment by joeyAghion** [comment]
This is 👍. Design- and scale-wise, it may make sense simply not to subscribe to channels for unchecked topics. But that can be for another day.

**Comment by ashkan18** [comment]
thanks @joeyAghion

**Pull request merged**